sql-plugin/src/main/311until320-all/scala/com/nvidia/spark/rapids/shims/ShuffleOriginUtil.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, NVIDIA CORPORATION.
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION.
```

Comment:
Copyright update with no other change?  Seems like this change should simply be reverted.